src/components/ChatEntry.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const id = props.id;
+  const sender = props.sender;
+  const body = props.body;
+  const time = TimeStamp({ time: props.timeStamp });
+  const liked = props.liked;
+  const handleLike = props.handleLike;
```

Comment:
I would recommend destructuring your props here too. This will keep your code a little cleaner. 🧹 😌 

For ex:
```
const ChatEntry = ({id, sender, body, time, liked, handleLike}) => {
```